### DISCUSSION TITLE
yank, comment and paste

### DISCUSSION THREAD

**Discussion by joe23rep** [numToStr/Comment.nvim#237] [Q&A] [answered]
just switched to comment.nvim and i really enjoy it. vast improvement over tcomment

however im missing one crucial keybinding which i cant get to work. i used to have a keybinding which bassically copies a line (or a selection) comments it out and then paste the uncommented part again. i use that quite regularly when i play around with color settings

that binding was `yy:TComment<CR>p`

i tried a few things going thew the plugin code but everything ive tried so far didnt work out.

maybe someone here can push me towards the right direction

**Comment by joe23rep** [comment, marked as answer]
wtf. just tried something i tried before and now it works all of a sudden.

i used `yy<Plug>(comment_toggle_linewise)<cr>p` and it works. i swear i tried exactly the same keybinding before and get the error tat i have trailing characters somewhere

sorry for bothering you. i somehow cant delete the post- only edit it.

**Reply by numToStr** [reply]
You might also want to read #70
